### PR TITLE
Added more unrank conditions

### PR DESCRIPTION
**Pull request by Rackover**
… game

I couldn't run the server on my side to test it, but : 
- It is valid python code (it runs)
- The modification is pretty small, so it shouldn't be any trouble

If you have a python server working on your side, feel free to test the PR !

### CODE REVIEW

**Comment by micheljung** [comment]
@Rackover will you write a unit test?

**Comment by Rackover** [comment]
I don't know how to write a unit test, but the first cool thing would be to simply be able to test it on a local python server. Which for now, is not possible. 😢

File: server/games/game.py [hunk 1]
```diff
@@ -80,7 +80,8 @@ class ValidityState(IntEnum):
     FFA_NOT_RANKED = 14
     UNEVEN_TEAMS_NOT_RANKED = 15
     UNKNOWN_RESULT = 16
-
+    MULTI_TEAM = 17
```
Comment (Sheeo) [inline server/games/game.py]: The database entries for these should be kept in sync

File: server/games/game.py [hunk 2]
```diff
@@ -199,6 +200,15 @@ def teams(self):
         return frozenset({self.get_player_option(player.id, 'Team')
                           for player in self.players})
 
+    @property
+    def is_multi_team(self):
+        teams = set()
```
Comment (Sheeo) [inline server/games/game.py]: Couldn't this simply be `return len(self.teams) > 2`?

**Comment by Sheeo** [comment]
We'll get you setup to test it locally; for now it's been out of sync with what's been going on in the db repo for a while, so it's hard to setup a local environment off the bat.

File: server/games/game.py (hunk 2 above)
Comment (Rackover) [reply server/games/game.py]: I didn't know this var existed. Well if we can use it for that, then sure.

File: server/games/game.py (hunk 1 above)
Comment (Rackover) [reply server/games/game.py]: you mean I must make a PR to the faf-db aswell ?

File: server/games/game.py (hunk 2 above)
Comment (Sheeo) [reply server/games/game.py]: It's defined immediately above :)

File: server/games/game.py (hunk 1 above)
Comment (Sheeo) [reply server/games/game.py]: Yes, as discussed in private.

**Comment by Rackover** [comment]
https://github.com/FAForever/db/pull/162

File: server/games/game.py [hunk 3]
```diff
@@ -199,6 +200,10 @@ def teams(self):
         return frozenset({self.get_player_option(player.id, 'Team')
                           for player in self.players})
 
+    @property
+    def is_multi_team(self):
+        return len(teams) > 2
```
Comment (Sheeo) [inline server/games/game.py]: Should be `len(self.teams) > 2`

**Comment by Rackover** [comment]
https://github.com/FAForever/server/pull/357

**Pull request closed without merging**